crypto/err/err_mark.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
The date here should be the same as was in err.c I think.

Reply:
These functions appeared in 2003, so I'll set that as starting year

Reply:
Done